components/AppNav.tsx

```diff
@@ -31,12 +31,14 @@ const AuthLink = () => (
     <NavLink
       path="https://github.com/garageScript/c0d3-app"
       className="nav-item nav-link"
+      external
```

Comment:
nice lol this error was bugging me

Reply:
Lol. I noticed that it only affects dev but not prod.